code/modules/reagents/chemistry/reagents.dm

```diff
@@ -87,6 +105,12 @@ GLOBAL_LIST_INIT(name2reagent, build_name2reagent())
 	if(material)
 		material = SSmaterials.GetMaterialRef(material)
 
+	if(acidpwr == 0)
```

Comment:
This should probably be an `isnull()` instead since someone might want to force a reagent with a high or low pH to have an acidpower of 0 anyway.